test/dummy/app/javascript/controllers/todo_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "@hotwired/stimulus"
+import { TurboConfirm } from "@rolemodel/turbo-confirm"
+
+export default class extends Controller {
+  connect() {
+    this.turboConfirm = new TurboConfirm(this.#config)
+  }
+
+  async confirmStatus({ currentTarget }) {
+    if (await this.turboConfirm.confirm()) {
+      currentTarget.innerHTML = 'Done'
+    } else {
+      currentTarget.innerHTML = 'Not Done'
+    }
+  }
```

Comment:
Of course the downside of the Promise rejection is this case becomes ugly. Unless you switch it back to promises:
```suggestion
  async confirmStatus({ currentTarget }) {
    this.turboConfirm.confirm()
      .then(() => { currentTarget.innerHTML = 'Done' })
      .catch(() => { currentTarget.innerHTML = 'Not Done' })
  }
```

```suggestion
  async confirmStatus({ currentTarget }) {
    try {
      await this.turboConfirm.confirm()
      currentTarget.innerHTML = 'Done'
    } catch {
      currentTarget.innerHTML = 'Not Done'
    }
  }
```

Reply:
@justwiebe I prefer resolving true/false.  I wish the Promise API made it more clear that reject is an exception.  Turbo also expects a true/false response.